src/probnum/randprocs/kernels/_kernel.py

```diff
@@ -342,6 +357,7 @@ def _evaluate(
         k_x0_x1 :
             See "Returns" section in the docstring of :meth:`__call__`.
         """
+        raise NotImplementedError
```

Comment:
I don't think we technically need this here, since the `ABC` can not be instantiated without implementing this method